ironfish/src/migrations/data/013-wallet-2/old/headers.ts

```diff
@@ -16,6 +18,229 @@ export type HeaderValue = {
   header: BlockHeader
 }
 
+class PartialBlockHeaderSerde {
```

Comment:
This is just copy and paste of previous one?

Reply:
Yep, needed to keep the migrations working.